src/components/team/Team.tsx

```diff
@@ -15,7 +16,8 @@ export default function Team(props: any) {
 
   // const imageWidth = Math.round(getViewportWidth() / 4);
 
-  const socialIconColor = '#580394';
+  // const socialIconColor = '#580394';
```

Comment:
Did it work, if so we can delete this line? And just bring along the comments above as well, they dont make sense anymore

Reply:
It didn't work for me.  The article said that CSS modules were what made the import to JS possible.  Are CSS modules turned on for this project?  I couldn't find a config file.

Reply:
Looking at the footer icons and the hero icons, they were all typed into the JS file, not brought in through sass.  I agree, it would be cool to get this to work so they could all reference the same variable!

Reply:
Yeah so I expect you to make sure it works before it is pushed into the repo. That you have to test locally before you add it to the PR, because i could have merged this and the it would not work :( So its important to have an insight to what is pushed to the PR :)